src/main/java/by/custom/utilcalculator/exception/UserFileNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package by.custom.utilcalculator.exception;
+
+import by.custom.utilcalculator.service.BundleResourcesServant;
+
+public class UserFileNotFoundException extends Exception {
```

Comment:
I think, it might be helpful if with and other exception contains more info. Like `file path`.

Reply:
I tried to add more info into exceptions, but I am not sure about success.